Automattic/newspack-ads
# feat: scaip placements

![image](https://user-images.githubusercontent.com/820752/142493536-9aea05bd-e671-4143-beb8-f66678a4b3fc.png)

Deprecate the use of SCAIP sidebars in favor of the new placements system. This PR creates placements based on the number of repetitions determined in SCAIP settings.

It is backwards compatible with existing Newspack Ad Unit widgets on SCAIP sidebars.

### How to test

1. On `master` branch, make sure you have Newspack Ad Unit widgets placed and with ad units selected on SCAIP sidebars
2. Switch to this branch, refresh your **Appearance -> Widgets** page and observe the sidebars are gone
3. Visit the Advertising -> Placements wizard and confirm the selected ad units are selected on the new UI
4. Visit a post and confirm that the ads also are also displayed as expected
5. Back to the Placements wizard, deactivate some placements and modify the ad unit for others and confirm the changes take place as expected
